src/energies.jl

```diff
@@ -6,14 +6,18 @@ struct Energies{T <: Number}
     # energies["TermName"]
     # parametrization on T acts as a nice check that all terms return correct type
     energies::OrderedDict{String, T}
+    total::T
```

Comment:
is this really needed? I'd rather avoid precomputation as far as possible

Reply:
No, actually I destroyed some functionality with the changes I made in the class. I'll undo it.